docs/src/pages/guides/examples.md

```diff
@@ -0,0 +1,246 @@
+
+```json
+{
+  "probes": [
+    {
+      "id": "1",
+      "name": "Probing Github",
+      "description": "simulate html form submission",
+      "interval": 10,
+      "requests": [
+        {
+          "method": "GET",
+          "url": "https://github.com/",
+          "timeout": 7000
+        },
+        {
+          "method": "GET",
+          "url": "https://github.com/hyperjumptech",
+          "timeout": 7000
+        }
+      ],
+      "incidentThreshold": 3,
+      "recoveryThreshold": 3,
+      "alerts": ["status-not-2xx", "response-time-greater-than-200-ms"]
+    }
+  ]
+}
+```
+
+In the configuration above, "Probing Github" probe will execute a GET request to `https://github.com`. If there are no triggered alerts such as `status-not-2xx` or `response-time-greater-than-200-ms`, it will execute a GET request to `https://github.com/hyperjumptech`.
+
+If there is a case where executing GET request to `https://github.com` triggered an alert, the next request will not be executed.
+
+## Requests Chaining
+
+Monika supports requests chaining, which enables you to do multiple request and use previous request(s) response for other request. For example, after executing a GET request to certain API, the next request could use the previous request(s) response into their path/query parameters or headers.
+
+Here is an example on how you could get previous request(s) response data into your next request:
+
+```
+{{ response.[0].status }} ==> Get status code from first request response
+{{ response.[1].data.token }} ==> Get token from second request response
+{{ response.[2].headers.SetCookie[0] }} ==> Get first cookie from third request response
+```
+
+Please refer to [Probe Response Anatomy](https://hyperjumptech.github.io/monika/guides/probes#probe-response-anatomy) in order to know which value could be used from the response body for the next request(s).
+
+### Pass Response Data as Path/Query Parameters
+
+Here is an example of passing previous request(s) response into the path/query parameters:
+
+```json
+{
+  "probes": [
+    {
+      "id": "1",
+      "name": "Probing Github",
+      "description": "simulate html form submission",
+      "interval": 10,
+      "requests": [
+        {
+          "method": "GET",
+          "url": "https://reqres.in/api/users",
+          "timeout": 7000
+        },
+        {
+          "method": "GET",
+          "url": "https://reqres.in/api/users/{{ responses.[0].data.data.[0].id }}",
+          "timeout": 7000
+        }
+      ],
+      "incidentThreshold": 3,
+      "recoveryThreshold": 3,
+      "alerts": ["status-not-2xx", "response-time-greater-than-2000-ms"]
+    }
+  ]
+}
+```
+
+In the configuration above, the first request will execute fetch all users available. If there are no triggered alerts, the response returned from the first request is ready to be used by the second request using values from `{{ responses.[0].data }}`. An example of the first request response should be like this:
```

Comment:
In the configuration above, the first request will fetch all users from `https://reqres.in/api/users`.  Then in the second request, Monika will fetch the details of the first user from the first request. If there are no triggered alerts, the response returned from the first request is ready to be used by the second request using values from `{{ responses.[0].data }}`.

Let's say the response from fetching all users in JSON format as follows: